lambdas/choice/src/writer.py

```diff
@@ -1,6 +1,78 @@
 import os
+import boto3
+
+"""
+Processes SNS messages containing user choices by writing choices to Dynamo.
+For event and database record shapes, see https://github.com/sjawhar/enchanted-brain/issues/4.
+"""
+
+
+EVENT_USER_ID_KEY = 'userId'
+EVENT_CHOICE_TYPE_KEY = 'choiceType'
+EVENT_CHOICE_KEY = 'choice'
+EVENT_TIMESTAMP_KEY = 'timestamp'
+
+CHOICE_COLOR = 'CHOICE_COLOR'
+CHOICE_EMOTION = 'CHOICE_EMOTION'
+CHOICE_CHILLS = 'CHOICE_CHILLS'
+CHOICE_IMAGERY = 'CHOICE_IMAGERY'
+
+CONCERT_SONG_ID = os.environ['EVENT_NAME']
+DYNAMODB_TABLE_NAME = os.environ['DYNAMODB_TABLE_NAME']
+
+choice_type_to_choice_key = {
+    CHOICE_COLOR: 'colors',
+    CHOICE_EMOTION: 'emotions',
+    CHOICE_CHILLS: 'chills',
+    CHOICE_IMAGERY: 'imagery'
+}
+choice_types = choice_type_to_choice_key.keys()
```

Comment:
`choice_types` is not being used. I'm going to add a linter and auto-formatter so we don't have to think about these things.